docs/src/pages/en/components/media-time-range.mdx

```diff
@@ -0,0 +1,165 @@
+## Preview thumbnails
+
+Adding the `<track default label="thumbnails" kind="metadata" src="">` 
+with a valid VTT file as `src` will enable the preview thumbnails on hover functionality.
+
+<ComponentSandpack
+  client:load
+  height={330}
+  html={`<media-controller>
+  <video 
+    playsinline muted crossorigin
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+  >
+    <track
+      default label="thumbnails" kind="metadata"
+      src="https://image.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/storyboard.vtt"
+    >
+  </video>
+  <media-control-bar>
+    <media-time-range></media-time-range>
+  </media-control-bar>
+</media-controller>`}
+/>
+
+
+
+## Custom current slot
+
+Add a time display component in the current slot that will slide along 
+the timeline as the video progresses.
+
+<ComponentSandpack
+  client:load
+  height={270}
+  html={`<media-controller>
+  <video
+    playsinline muted crossorigin
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+  ></video>
+  <media-control-bar>
+    <media-time-range>
+      <media-time-display slot="current"></media-time-display>
+    </media-time-range>
+  </media-control-bar>
+</media-controller>`}
+/>
+
+
+
+## Remove preview elements
+
+Set the `preview` slot to an empty element to remove the default preview elements.
+
+<ComponentSandpack
+  client:load
+  height={270}
+  html={`<media-controller>
+  <video
+    playsinline muted crossorigin
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+  ></video>
+  <media-control-bar>
+    <media-time-range>
+      <span slot="preview"></span>
+    </media-time-range>
+  </media-control-bar>
+</media-controller>`}
+/>
+
+
+
+## Styling
+
+### Using CSS variables
+
+The `<media-time-range>` consists of several elements including an 
+[`<input type="range">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) 
+element which can be tricky to style across browsers but the `<media-time-range>` component
+comes with targeted CSS variables that make this a breeze.
+
+<ComponentSandpack
+  client:load
+  height={270}
+  html={`<style>
+  media-time-range {
+    --media-control-background: transparent;
+    --media-control-hover-background: transparent;
+    --media-range-track-border-radius: 3px;
+    --media-range-thumb-opacity: .7;
+    --media-preview-time-margin: 0 0 -15px;
+  }
+</style>
+<media-controller>
+  <video
+    playsinline muted crossorigin
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+  ></video>
+  <media-control-bar>
+    <media-time-range></media-time-range>
+  </media-control-bar>
+</media-controller>`}
+/>
+
+
+Discover all the available [CSS variables](#css-variables).
+
+### Using attributes
+
+The `<media-time-range>` will be updated with **media state** attributes any time 
```

Comment:
we should be consistent in what we're calling these and use those everywhere. The section is called media ui attributes, so, should we call it media ui attributes here instead of media state attributes?

Reply:
good call, I'll change it to media UI attributes